src/config.js

```diff
@@ -1,19 +1,22 @@
 const baseUrl = 'https://services.arcgis.com/ZzrwjTRez6FJiOq4/arcgis/rest/services';
 export default {
+  notProd: process.env.REACT_APP_ENVIRONMENT !== 'production',
   urls: {
     baseUrl,
     hazardGroupingsTable: `${baseUrl}/Report_Tables_View/FeatureServer/0`,
+    hazardGroupTextTable: `${baseUrl}/Report_Tables_View/FeatureServer/1`,
     hazardIntroTextTable: `${baseUrl}/Report_Tables_View/FeatureServer/2`,
     hazardReferenceTextTable: `${baseUrl}/Report_Tables_View/FeatureServer/3`,
-    hazardUnitTextTable: `${baseUrl}/Report_tables_View/FeatureServer/4`
+    hazardUnitTextTable: `${baseUrl}/Report_Tables_View/FeatureServer/4`,
+    otherDataTable: `${baseUrl}/Report_Tables_View/FeatureServer/7`,
+    reportTextTable: `${baseUrl}/Report_Tables_View/FeatureServer/8`
   },
   queries: [
-    // ['Utah_Geologic_Hazards/FeatureServer/0', '???'], // flood canyon
+    ['Utah_Geologic_Hazards/FeatureServer/0', 'FCH'],
     ['Utah_Geologic_Hazards/FeatureServer/1', 'FLH'],
     ['Utah_Geologic_Hazards/FeatureServer/2', 'SGS'],
     ['Utah_Geologic_Hazards/FeatureServer/3', 'LSS'],
     // ['Utah_Geologic_Hazards/FeatureServer/4', '???'], // landslides
```

Comment:
https://github.com/agrc/ugs-hazards/issues/2#issuecomment-542264075

```suggestion
    ['Utah_Geologic_Hazards/FeatureServer/4', 'LSF'], // landslides
```